lib/katello/repository_types/python.rb

```diff
@@ -0,0 +1,16 @@
+require 'pulp_python_client'
+
+Katello::RepositoryTypeManager.register('python') do
+  allow_creation_by_user true
+  pulp3_service_class Katello::Pulp3::Repository::Generic
+  pulp3_api_class Katello::Pulp3::Api::Generic
+  pulp3_plugin 'python'
+  partial_repo_path '' #TODO: add partial repo path
+
+  repositories_class PulpPythonClient::RepositoriesPythonApi
```

Comment:
I realized in some cases you have `api` in the variable name and sometimes you don't. I think it would be best to stick to a single convention (in this case probably keeping `api` in there).

Reply:
Edit: we should put API when the Pulp class has API in it.